### DISCUSSION TITLE
Remove spinors on an amplitude generated by FCFAConvert

### DISCUSSION THREAD

**Discussion by JulRovn** [FeynCalc/feyncalc#221] [Q&A]
Hello,
I am working with a four-fermion operator and when using the command FCFAConvert and selecting Truncated->True, there is a problem with the DiracJoinChain
That is why I am working setting Truncated->True and then I would like to remove the spinors. 
What is the command I should write for FeynArts and/or FeynCalc?
Since I am working at an operator level, I also have want to remove some gamma matrices and SU(3). Is it the same command? 
Thank you in advance

**Comment by vsht** [comment]
Hi,

removing the spinors can be tricky because that leaves you with open Dirac chains
and this is something that FeynCalc is not particularly good at. I guess something like this should work

```
sps = Cases2[amp, Spinor]
dis = FCMakeSymbols[di, Range[1, Length[sps]], List]
ru = Thread[Rule[sps, dis]]
amp // FCDiracIsolate[#, Head -> dch] & //  ReplaceAll[#,  dch[DOT[a_Spinor, b___, c_Spinor]] :> DCHN[b, a /. ru, c /. ru]] &
```
> Since I am working at an operator level, I also have want to remove some gamma matrices and SU(3). Is it the same command?

If you are new to FeynCalc, there are [tutorials](https://github.com/FeynCalc/feynknowledge/tree/master/Tutorials) and a [manual](https://github.com/FeynCalc/feyncalc-manual/releases/tag/dev-manual) that one should go through first. A good familiarity with Mathematica
programming is also essential here. Sorry, but FeynCalc is not as simple to use as Madgraph & co. and it requires some effort to master the package.

**Reply by Show-Down** [reply]
Hi,
I want to know if I could remove the external vector lines in some Feynman diagrams such as tree diagram of photon gluon fusion? thamks.

**Reply by vsht** [reply]
You'd need some custom code for that.

**Reply by Show-Down** [reply]
I import the StandardForm and substitute what I want
